default.nix

```diff
@@ -119,10 +119,13 @@ rec {
       '';
       doCheck = true;
       checkPhase = ''
-        test -n "$(find $out/haskell/ -type f -name '*.hs')"
+        test -n "$(find $out/ -type f -name '*.hs')"
```

Comment:
I see that the Makefile still produces the Haskell source code under `$out/haskell/`, so does that mean that checking happens after `install`? In which case this block (`doCheck`+`checkPhase`) should be moved right after `installPhase`.

Reply:
I think checking happens after building and before installing.

Reply:
Nix is declarative, so rearranging doesn't affect anything here.

Reply:
So then this change seems redundant right? (since `$out/haskell` will indeed contain the generated `*.hs` files)

Reply:
yeah, you're right, I must've accidentally left this in